components/message_submit_error.jsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {isErrorInvalidSlashCommand} from 'utils/post_utils.jsx';
+
+class MessageSubmitError extends React.PureComponent {
+    static propTypes = {
+        error: PropTypes.object.isRequired,
+        handleSubmit: PropTypes.func.isRequired,
+        submittedMessage: PropTypes.string,
+    }
+
+    renderSlashCommandError = () => {
+        if (!this.props.submittedMessage) {
+            return this.props.error.message;
+        }
+
+        const command = this.props.submittedMessage.split(' ')[0];
+        return (
+            <FormattedMessage
+                id='message_submit_error.invalidCommand'
+                defaultMessage={'Command with a trigger of \'{command}\' not found. {send_as_message}'}
+                values={{
+                    command,
+                    send_as_message: (
```

Comment:
Question here:
Should we keep this as part of another format message statement?
Instead of something having them as two separate messages independent of each other?
```
<React.Fragment>
      <FormattedMessage
           id='message_submit_error.invalidCommand'
            ... 
      </FormattedMessage> 
      <a
           ref='sendAsMessageLink'
           href='#'
           onClick={this.props.handleSubmit}
           className='message-submit-error__send-as-message'
        >
            <FormattedMessage
                id='message-submit-error.sendAsMessageLink'
                 ... 
           </FormattedMessage> 
       </a>
</React.Fragment>

```

We are trying to avoid HTML/components as part of `en.json` so, if this does not have to be nested would you mind changing this `FormatMessage` to be independent of the previous one?